.browserslistrc

```diff
@@ -0,0 +1,10 @@
+last 2 Chrome versions
+IE >=11
+last 2 Firefox versions
+Safari >=5.1
+Opera >=15
+iOS >=6.1
+Android >=4.1
+
+IE 10 # only basic support according to the mw browser compatibility matrix.
+# TODO: how to separate CSS and JS browser support?
```

Comment:
We could certainly pimp the `postcss.config.js` to merge `.browserslistrc` and the CSS-only browser list but natively I could not find the hybrid version [at first glance](https://github.com/browserslist/browserslist#user-content-queries).